examples/simple_script/BUILD

```diff
@@ -30,3 +31,28 @@ ruby_test(
         "@bundle//:rspec",
     ],
 )
+
+
+ruby_binary(
+    name = "rubocop",
+    main = "@bundle//:bin/rubocop",
```

Comment:
This is really cool — being able to reference binaries like that! ✅

Do you think it makes sense to create a standalone `ruby_rubocop` Bazel command?

Reply:
we will, but best get the basics working first